assets/product-compare.scss.liquid

```diff
@@ -0,0 +1,146 @@
+
```

Comment:
I'm not a huge fan of this duplication if we needed to change the breakpoints we'd have to sift through all the files and change them wherever we duplicated them. Probably just pull this into the main SCSS so you can reference the variables. If we need to split out the style files we can look at creating a variables file that is shared and imported but we have bigger performance fish to fry.